### PR TITLE
Added routes for Comment

### PR DESCRIPTION
**Pull request by sourabh1031**
Please review it

### CODE REVIEW

**Comment by sr6033** [comment]
Where are the routes?
A blank route means a route which is present and having options to be added to for future implementation.

**Comment by sourabh1031** [comment]
@sr6033 Please review it now

**Comment by sakshamsaxena** [comment]
There's no need to create a separate class for Comments. It's not the optimal design in our case. Your newly added code should rather be a part of Blog itself. See carefully how the comment object is related to a blog post.

**Comment by sourabh1031** [comment]
@sakshamsaxena  @sr6033  Please review it  now

**Comment by sourabh1031** [comment]
@sakshamsaxena  review it now

File: routes/Blog.js [hunk 1]
```diff
@@ -125,4 +125,12 @@ BlogPost.patch('/id/:id/dislike', function(req, res) {
...
 });
 
+BlogPost.patch('/id/:id/comment/:cid/like', function(req, res) {
+	res.send('/Blog' + req.url);
+});
+
+BlogPost.patch('/id/:id/comment/:cid/dislike', function(req, res) {
+	res.send('/Blog' + req.url);
+});
+
```
Comment (sr6033) [inline routes/Blog.js]: @sakshamsaxena Have a look now.

Comment (sakshamsaxena) [reply routes/Blog.js]: Great! Add a bit of inline documentation like the rest of the routes as well. Then it'll be good to merge

**Comment by sourabh1031** [comment]
@sr6033 @sakshamsaxena i have added the necessary documentation please review it

**Comment by sakshamsaxena** [comment]
@sourabh1031 Merged! Thanks!

**Comment by sourabh1031** [comment]
@sakshamsaxena i must say thanks to you and @sr6033 you both helped me to hack it before the deadline

**Pull request merged**